.circleci/config.yml

```diff
@@ -487,6 +503,51 @@ jobs:
           replace: $SNAPSHOT_VERSION
       - notify_github:
           data: "$(jq -Rcs '{ body: . }' scripts/snapshot-comment.md)"
+  build_check_android:
+    parameters:
+      working_directory:
+        type: string
+      android_path:
+        type: string
```

Comment:
I feel like `android_path` is always `<<parameters.working_directory>>/android`, do we need to pass it as a parameter as well?